### PR TITLE
Move requirements.txt in lmnet to the top

### PR DESCRIPTION
**Pull request by iizukak**
## Motivation and Context

To create blueoil python package in the furture, We want to move requirements.txt to the top directory.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -91,8 +91,8 @@ RUN apt-get update && apt-get install -y g++-5-arm-linux-gnueabihf && \
     ln -s /usr/bin/arm-linux-gnueabihf-g++-5 /usr/bin/arm-linux-gnueabihf-g++
 
 # Install requirements
-COPY lmnet/*requirements.txt /tmp/requirements/
-RUN pip install -r /tmp/requirements/gpu.requirements.txt
+COPY ./*requirements.txt /home/blueoil/
```
Comment (suttang) [inline docker/Dockerfile]: I think we don't need to copy requirement to `/home/blueoil`.
requirements.txt is not required for production use.

Comment (hadusam) [reply docker/Dockerfile]: +1

Comment (iizukak) [reply docker/Dockerfile]: @suttang 
We need `requirements.txt` in Docker image because we need to install requirements.
Which path is good ?

Comment (suttang) [reply docker/Dockerfile]: @iizukak 
/tmp/requirements/ would be better.

**Comment by iizukak** [comment]
@suttang Oh, I simply misunderstood. Path was fixed.

File: lmnet/tox.ini [hunk 2]
```diff
@@ -7,7 +7,7 @@ skipsdist = True
 [testenv]
 recreate = True
 deps =
-    -r{toxinidir}/gpu.requirements.txt
+    -r{toxinidir}/../gpu.requirements.txt
```
Comment (iizukak) [inline lmnet/tox.ini]: @suttang @hadusam 
In docker image, we have requreiments.txt in `/tmp/requirements/`, When tox executed, these files cannot find requirements.txt. Do you have any idea to find path in tox using docker?

Comment (suttang) [reply lmnet/tox.ini]: IMO tox command via docker will not be used much.
If you want to run via docker, you just use -v.

Comment (iizukak) [reply lmnet/tox.ini]: >  tox command via docker will not be used much.

`make test-lmnet` use tox. So this line cause CI failure.
Thinking time...

**Comment by iizukak** [comment]
Talking with @suttang and @hadusam, We decide to put `*requirements.txt` in `/home/blueoil` for tox issue.

**Pull request merged**